src/utils/constants.ts

```diff
@@ -27,12 +35,20 @@ export const COLOR_DARK_RED = '#730413';
 export const COLOR_PRIMARY_BLUE = '#011e41';
 export const COLOR_PRIMARY_RED = '#f5333f';
 
-export const OPERATION_TYPE_PROGRAMME = 0;
-export const OPERATION_TYPE_EMERGENCY = 1;
+export const OPERATION_TYPE_PROGRAMME = 0 satisfies OperationTypeEnum;
+export const OPERATION_TYPE_EMERGENCY = 1 satisfies OperationTypeEnum;
+// FIXME: Remove this multi thing
 export const OPERATION_TYPE_MULTI = -1;
 
 export const CATEGORY_RISK_VERY_LOW = 1;
 export const CATEGORY_RISK_LOW = 2;
 export const CATEGORY_RISK_MEDIUM = 3;
 export const CATEGORY_RISK_HIGH = 4;
 export const CATEGORY_RISK_VERY_HIGH = 5;
```

Comment:
Doesn't category have any ENUM?

Reply:
maybe it does

Reply:
Then we need to add "satisfies" to these values as well.